config/disabled.yml

```diff
@@ -68,6 +68,10 @@ Style/DocumentationMethod:
     - 'spec/**/*'
     - 'test/**/*'
 
+Style/EmptyLineAfterGuardClause:
+  Description: 'Add empty line after guard clause.'
+  Enabled: false
```

Comment:
@bbatsov @unkmas This cop seems to be useful, so I think that it can be enabled by default. And only adding blank lines, it is also preferable that `git blame` does not become dirty. What do you think about this?

Reply:
@koic I disabled it by default because I don's sure that it is common style of coding. I asked my friends and only half of them uses it.

If you think, that is should be enabled - then great, I will be glad to enable it :)

Reply:
@koic your thumbsup means "yeah, enable it" or "yeah, it should not be enabled"? :)

Reply:
For my personal opinion, I hope that it can be enable. In addition, I'd like to ask the opinion of the core developers.

Reply:
@bbatsov What do you think about it?

Reply:
I have been waiting for this cop for a long time (and also thinking about implementing it myself), but I would really like to see it be `EmptyLinesAroundGuardClause`. Only enforcing after the guard clause is not useful to me.